include/openPMD/IO/AbstractIOHandler.hpp

```diff
@@ -56,6 +56,28 @@ class unsupported_data_error : public std::runtime_error
     virtual ~unsupported_data_error() { }
 };
 
+/**
+ * @brief Determine what items should be flushed upon Series::flush()
+ *
+ */
+enum class FlushLevel : unsigned char
+{
+    UserFlush,
```

Comment:
Can you also add a `UserFlush` doc string, please?

Reply:
I need to rename some of those anyway, planned to do that then.